### PR TITLE
Adding Playlab characters to stickers.

### PR DESCRIPTION
**Pull request by stevekrenzel**
This change adds the playlab characters to the stickers block. It
also resizes the existing sticker images to be the same size as the
playlab images.

### CODE REVIEW

File: apps/src/turtle/skins.js [hunk 1]
```diff
@@ -39,38 +39,50 @@ exports.load = function (assetUrl, id) {
...
+
+    var mapping = {};
+
+    var i, name;
+    for(i = 0; i < playlab.length; i++) {
+      name = playlab[i];
+      mapping[name] =  assetUrl('media/skins/studio/' + name.toLowerCase() + '_thumb.png');
+    }
+
+    for(i = 0; i < misc.length; i++) {
```
Comment (Bjvanminnen) [inline apps/src/turtle/skins.js]: style nit: space after `for` here and line 66

File: apps/src/turtle/skins.js [hunk 2]
```diff
@@ -39,38 +39,50 @@ exports.load = function (assetUrl, id) {
...
+
+    // Miscellaneous stickers
+    var misc = ['Beaver', 'Bunny'      , 'Chicken', 'Elephant', 'Giraffe',
+                'Goat'  , 'Grasshopper', 'Hippo'  , 'Lion'    , 'Llama'  ,
+                'Monkey', 'Moose'      , 'Mouse'  , 'Owl'     , 'Peacock',
+                'Rocket', 'Triceratops', 'Turtle' , 'Zebra'   ];
+
+    var mapping = {};
+
+    var i, name;
```
Comment (stevekrenzel) [inline apps/src/turtle/skins.js:65]: These are defined outside of the loop to make it clear that they are intended to be used in multiple "scopes" (in this loop and the next)

File: apps/src/turtle/skins.js (hunk 1 above)
Comment (stevekrenzel) [inline apps/src/turtle/skins.js]: Fixed.

**Comment by Bjvanminnen** [comment]
LGTM :)

**Pull request merged**